### PR TITLE
[GITHUB-48] Make workdir path fully configurable

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

**Review by nathanwebsterdotme — APPROVED** [review]

File: meta/main.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ---
 
 galaxy_info:
+  author:
```
Comment (lobsterdore) [inline meta/main.yml]: Let's just do ```author: sansible``` - much easier to maintain :)

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,5 @@
 docker-py
+flake8
 molecule
+yamllint
+ansible-lint
```
Comment (dennisconrad) [inline requirements.txt]: Can we peg the proper modules and versions, please?

docker==3.5.1
molecule==2.19.0

Comment (wojtek-oledzki) [reply requirements.txt]: updated with scaffold :)

**Review by dennisconrad — APPROVED** [review]

**Comment by lobsterdore** [comment]
LGTM 👍

**Pull request merged**